### ISSUE TITLE
Sort packages alphanumerically

### ISSUE THREAD

**Issue by MoritzWeber0** [hadolint/hadolint#976]
- [ ] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

In the Dockerfile best-practises, it is recommended to sort multi-line arguments alphanumerically: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

While it is difficult to implement this in a general way for all commands, it could be enforced for package installations via package managers.

Let's take this minimal Dockerfile as example: 

```Dockerfile
FROM debian:bookworm-slim

RUN apt-get update && \
    apt-get install --no-install-recommends --yes \
    vim=2:9.0.1378-2 \
    python3=3.11.2-1+b1 \
    && rm -rf /var/lib/apt/lists/*
```

It should raise `[hadolint] info: Sort apt-get packages alphanumerically (DLXXXX)`. 

A valid Dockerfile without warning or info would be: 

```Dockerfile
FROM debian:bookworm-slim

RUN apt-get update && \
    apt-get install --no-install-recommends --yes \
    python3=3.11.2-1+b1 \
    vim=2:9.0.1378-2 \
    && rm -rf /var/lib/apt/lists/*
```

Here, the package list is properly sorted: 
```py
> sorted(['vim', 'python3'])
['python3', 'vim']
```

### Actual behavior

There is no such rule yet, as far as I can see.
